core/src/test/scala/integration/kafka/api/AdminClientTest.scala

```diff
@@ -106,7 +106,7 @@ class AdminClientTest extends IntegrationTestHarness with Logging {
 
     val consumerSummaries = client.describeConsumerGroup(groupId)
     assertEquals(1, consumerSummaries.size)
-    assertEquals(Set(tp, tp2), consumerSummaries.head.assignment.toSet)
+    assertEquals(Set(tp, tp2), consumerSummaries.get.head.assignment.toSet)
```

Comment:
@ijuma not sure what you meant by None return case since the None case is prevalent in ConsumerGroupCommand.scala in the describeGroup function but this file is testing the nonexistentgroup and existent group scenarios via the describeConsumerGroup function in AdminClient.scala